armi/reactor/assemblies.py

```diff
@@ -814,6 +812,31 @@ def dump(self, fName=None):
         with open(fName, "w") as pkl:
             pickle.dump(self, pkl)
 
+    def iterBlocks(self, typeSpec=None, exact=False):
```

Comment:
So, why is `iterBlocks()` necessary? For an `Assembly` this is the same as iterating through all the children (shallow, not deep), right?

Reply:
You're totally right. I added this because I wanted to do a simple find/replace `Assembly.getBlocks` -> `Assembly.iterBlocks` in cases where it made sense. But I could just as easily replace that with `iterChildren` or `iterChildrenWithFlags` as appropriate. Then we have one less method floating around

Reply:
I was just thinking of "The Zen of Python":

> There should be one-- and preferably only one --obvious way to do it.

It is easier on users if the API is smaller.

BUT I won't die on this hill.